Library/Homebrew/download_strategy.rb

```diff
@@ -538,6 +538,8 @@ class CurlGitHubPackagesDownloadStrategy < CurlDownloadStrategy
   private
 
   def _fetch(url:, resolved_url:)
+    checksum ||= url[/sha256:([0-9a-fA-F]{64})$/, 1]&.downcase
```

Comment:
This regex is duplicated, would be good to unify it. Also note this is going to require rebasing after https://github.com/Homebrew/brew/pull/10977 and taking another approach.

Reply:
Done